README.md

```diff
@@ -23,7 +23,7 @@ You can download release builds through the [releases section of this](https://g
   <dependency>
     <groupId>software.amazon.payloadoffloading</groupId>
     <artifactId>payloadoffloading-common</artifactId>
-    <version>1.0.0</version>
+    <version>2.0.0</version>
```

Comment:
Sorry about that, but can we also leave the v1, so we would have something like
### Version 2.x
  ``` xml
  <dependency>
      <groupId>...</groupId>
      <artifactId>...</artifactId>
      <version>2.0.0</version>
  </dependency>
  ```

### Version 1.x
  ``` xml
  <dependency>
      <groupId>...</groupId>
      <artifactId>...</artifactId>
      <version>1.0.0</version>
  </dependency>
  ```

Reply:
done